coral/src/app/features/approvals/components/RequestDetailsModal.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import RequestDetailsModal from "src/app/features/approvals/components/RequestDetailsModal";
+
+const baseProps = {
+  onClose: jest.fn(),
+  onApprove: jest.fn(),
+  onReject: jest.fn(),
+};
+
+describe.only("RequestDetailsModal.test", () => {
+  afterEach(cleanup);
+
+  it("renders a Modal with correct elements (isLoading false)", () => {
+    render(
+      <RequestDetailsModal {...baseProps} isLoading={false}>
+        <div>content</div>
+      </RequestDetailsModal>
+    );
+
+    expect(
+      screen.getByRole("heading", { name: "Request details" })
+    ).toBeVisible();
+    expect(screen.getByText("content")).toBeVisible();
+    expect(screen.getByRole("button", { name: "Close modal" })).toBeVisible();
+    expect(screen.getByRole("button", { name: "Close modal" })).toBeVisible();
+    expect(screen.getByRole("button", { name: "Approve" })).toBeEnabled();
+    expect(screen.getByRole("button", { name: "Approve" })).toBeEnabled();
+    expect(screen.getByRole("button", { name: "Reject" })).toBeVisible();
```

Comment:
`.toBeEnabled()` makes `.toBeVisible()` obsolete, since it can't be enabled if it's not visible! 

And I always use `toBeEnabled` instead of `toBeVisible` for elements where that's possible, because it's a more useful assertion (it's not only there, it's also a functioning button) and `toBeDisabled` in when it's not supposed to be enabeld.

Reply:
[9a86ee0](https://github.com/aiven/klaw/pull/611/commits/9a86ee0ef5928940fafc36fa55025ed02bd64324)